server/repositories/RawCaptureRepository.js

```diff
@@ -7,10 +9,88 @@ class RawCaptureRepository extends BaseRepository {
     this._session = session;
   }
 
+  _filterWhereBuilder(object, builder) {
+    const result = builder;
+    const {
+      whereNulls = [],
+      whereNotNulls = [],
+      whereIns = [],
+      ...parameters
+    } = object;
+
+    if (parameters.tokenized === 'true') {
+      whereNotNulls.push('wallet.token.id');
+    } else if (parameters.tokenized === 'false') {
+      whereNulls.push('wallet.token.id');
```

Comment:
if we need to communicate with the wallet schema then this should be in the query service